pkg/services/featuremgmt/toggles_gen.json

```diff
@@ -0,0 +1,2013 @@
+{
+  "kind": "FeatureList",
+  "apiVersion": "featuretoggle.grafana.app/v0alpha1",
+  "metadata": {},
```

Comment:
^^ not saving the max `resourceVersion` because that would likely lead to dumb git merge conflicts